src/main/java/com/Leapwork/Leapwork_plugin/PluginHandler.java

```diff
@@ -246,7 +248,7 @@ public String getWorkSpaceSafe(FilePath workspace, EnvVars env)
                 default:
                     StringBuilder errorMessage = new StringBuilder(String.format(Messages.ERROR_CODE_MESSAGE, response.getStatusCode(), response.getStatusText()));
                     OnFailedToGetScheduleTitleIdMap(null,errorMessage.toString(),listener);
-
+                    throw new Exception(errorMessage.toString());
```

Comment:
`OnFailedToGetScheduleTitleIdMap` is throwing an exception already, check the code

Reply:
removed